core/blockchain.go

```diff
@@ -2,6 +2,7 @@ package core
 
 import (
 	"errors"
+	"fmt"
```

Comment:
I assume this change was accidental/automatic. Could you drop it out? Don't want to touch more files than necessary, to minimize conflict sets

Reply:
Thanks for the catch